packages/ui/src/components/form-input/index.ts

```diff
@@ -1,24 +1,23 @@
-import { FormCheckbox, type FormCheckboxPropsType } from './components/form-checkbox'
```

Comment:
can we delete all the `form-x` components now? I wonder how much cleanup this will result in.
wish we had tests for this to verify easily if everything works as expected!